authors/apps/profiles/models.py

```diff
@@ -0,0 +1,48 @@
+from django.db import models
+from django.dispatch import receiver
+from django.db.models.signals import post_save
+from authors.settings import AUTH_USER_MODEL
+from cloudinary.models import CloudinaryField
+
+
+class Profile(models.Model):
+    """
+    This model creates a user profile with bio and
+    image field once a user creates an account.
+    """
+    user = models.OneToOneField(AUTH_USER_MODEL, on_delete=models.CASCADE)
+    bio = models.TextField(default="Update your bio description")
+    image = CloudinaryField('image')
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    @property
+    def image_url(self):
+        """
+        Get the image url. This spits a url that resolves to the image.
+        """
+        return self.image.url
+
+    @property
+    def username(self):
+        return self.user.username
+
+    def __str__(self):
+        return self.user.username
+
+
+@receiver(post_save, sender=AUTH_USER_MODEL)
+def create_user_profile(sender, instance, created, **kwargs):
```

Comment:
Using signals will work but on some research, it is not among the best practices in django [link](https://lincolnloop.com/blog/django-anti-patterns-signals/)
The intended functionality here can be achieved in the `RegistrationSerializer` create method.